src/main/kotlin/com/lovelysystems/gradle/LSAws.kt

```diff
@@ -74,10 +74,8 @@ fun Project.awsProject(profile: String, ssoSessionSettings: SsoSessionSettings)
                          - Use `${SsoSessionSettings.NAME}` when prompted for the 'SSO session name', for the rest use the default values.
                         """.trimIndent()
                         error(msg)
-                    } else if (error.ssoTokenError) {
```

Comment:
note: just tried again and removed this explicit handling as there can be multiple different error message I just saw. not really needed in the end and we can always try to login